modules/xfem/src/constraints/XFEMEqualValueAtInterface.C

```diff
@@ -70,9 +82,13 @@ XFEMEqualValueAtInterface::computeQpResidual(Moose::DGResidualType type)
   return r;
 }
 
+template <bool is_ad>
 Real
-XFEMEqualValueAtInterface::computeQpJacobian(Moose::DGJacobianType type)
+XFEMEqualValueAtInterfaceTempl<is_ad>::computeQpJacobian(Moose::DGJacobianType type)
 {
+  if (is_ad)
+    return 0;
```

Comment:
this should never get called right? I would error for clarity for other developers looking at this